Build/windows/generate.bat

```diff
@@ -0,0 +1,108 @@
+:: Copyright(c) 2018 Intel Corporation
+:: SPDX-License-Identifier: BSD-2-Clause-Patent
+@echo off
+
+setlocal
+cd /d "%~dp0"
+set instdir=%CD%
+
+if -%1-==-- call :help
+call :args %*
+call :buildCmake
+goto :EOF
+
+:args
+if -%1-==-- (
+    goto argend
+) else if "%1"=="/help" (
+    call :help
+) else if "%1"=="help" (
+    call :help
+) else if "%1"=="clean" (
+    echo Cleaning build folder
+    for %%i in (*) do if not "%%~i" == "generate.bat" del "%%~i"
+    for /d %%i in (*) do if not "%%~i" == "generate.bat" (
+        del /f /s /q "%%~i" 1>nul
+        rmdir /s /q "%%~i" 1>nul
+    )
+    exit
+) else if "%1"=="2019" (
+    echo Building for Visual Studio 2019
+    set "GENERATOR=Visual Studio 16 2019"
+    shift
+) else if "%1"=="2017" (
+    echo Building for Visual Studio 2017
+    set "GENERATOR=Visual Studio 15 2017 Win64"
+    shift
+) else if "%1"=="2015" (
```

Comment:
Do we need to support vs 2008-2013? How about 2015 and later?

Reply:
I've put those in for now since I do not know what configurations are supported. I can mark those as unsupported

Reply:
Yeah, please mark those unsupported for now, since those are not tested.